src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts

```diff
@@ -942,6 +944,15 @@ var requirejs = (function() {
 			this._sendMessageToWebview({ ...inset.cachedCreation, initiallyHidden: this.hiddenInsetMapping.has(output) });
 		}
 
+		if (this.firstInit) {
+			// On first run the contents have already been initialized so we don't need to init them again
+			this.firstInit = false;
```

Comment:
This is fine but seems like it would be better if something in `initializeMarkup` could tell whether the initialization had definitely already happened. Otherwise seems like we might make a change somewhere else and hit this block. Not super familiar with this code though.